Doc/library/stdtypes.rst

```diff
@@ -2031,38 +2255,69 @@ expression support in the :mod:`re` module).
 
    Return the highest index in the string where substring *sub* is found, such
    that *sub* is contained within ``s[start:end]``.  Optional arguments *start*
-   and *end* are interpreted as in slice notation.  Return ``-1`` on failure.
+   and *end* are interpreted as in slice notation::
+
+      >>> 'spam, spam, spam'.rfind('sp')
+      12
+      >>> 'spam, spam, spam'.rfind('sp', 0, 10)
+      6
 
 
+   Return ``-1`` on failure.
+   See also :meth:`find`.
+
 .. method:: str.rindex(sub[, start[, end]])
 
    Like :meth:`rfind` but raises :exc:`ValueError` when the substring *sub* is not
-   found.
+   found::
 
+      >>> 'spam, spam, spam'.rindex('eggs')
+      Traceback (most recent call last):
+        File "<stdin>", line 1, in <module>
+      ValueError: substring not found
+
+   See also :meth:`index`.
 
 .. method:: str.rjust(width[, fillchar])
 
    Return the string right justified in a string of length *width*. Padding is
-   done using the specified *fillchar* (default is an ASCII space). The
-   original string is returned if *width* is less than or equal to ``len(s)``.
+   done using the specified *fillchar* (default is an ASCII space)::
 
+      >>> 'Python'.rjust(10)
+      '    Python'
+      >>> 'Python'.rjust(10, '.')
+      '....Python'
+
+    The original string is returned if *width* is less than or equal to ``len(s)``.
```

Comment:
I think this might fix the [Sphinx error](https://github.com/python/cpython/actions/runs/9390730454/job/25861276052?pr=119445):

```suggestion
   The original string is returned if *width* is less than or equal to ``len(s)``.
```

For some reason, reST is usually indented in multiples of 3 spaces.